tests/order_with_respect_to/tests.py

```diff
@@ -10,13 +10,19 @@
 
 class OrderWithRespectToTests(TestCase):
 
+    # Provide these models as class attributes so that we can subclass this
```

Comment:
I would simply say
`# Hook for subclasses to run these tests with alternate models.`
(details about which other classes use it may go stale)